subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -353,6 +358,17 @@ bool pe_dpm_initiated_ams(const struct device *dev)
 	return atomic_test_bit(pe->flags, PE_FLAGS_DPM_INITIATED_AMS);
 }
 
+/**
+ * @brief End and atomic messaging sequence
+ */
+void pe_dpm_end_ams(const struct device *dev)
```

Comment:
Right now, you set this flag directly and sometimes clear it directly. It would be nice to handle consistently via these accessors.

Reply:
I've made the change for clearing the bit. The other accessors aren't used in this file.